username_api.py

```diff
@@ -110,4 +158,4 @@ def check(website, username):
 	return jsonify(check_username(website, username))
 
 if __name__ == '__main__':
-	app.run(host='0.0.0.0', port=8521)
\ No newline at end of file
+	app.run(host='0.0.0.0', port=8521)
```

Comment:
Unrelated change, #41 will take care of this.

Reply:
Well, none of these are conflicting/breaking changes, I guess it's fine for them to be here as well.

Reply:
If you're gonna keep them here, you should do them in a new commit so there's not a bunch of random changes in a commit that adds a new feature 👍

Reply:
Lol, it's just a newline at EOF, nonetheless, I've undone those changes.

Reply:
Yea, sorry for being nitpicky, it's just generally distracting from the purpose of the PR, and makes the `git blame` look weird.